locale/en_US/submission.po

```diff
@@ -1475,6 +1478,9 @@ msgstr "Revision required."
 msgid "notification.type.editorDecisionResubmit"
 msgstr "Resubmit for review."
 
+msgid "notification.type.editorDecisionNewRound"
+msgstr "New review round."
```

Comment:
> A new round of reviews was initiated.

Reply:
fixed

Reply:
Probably no need to restarts tests for this?

Reply:
Nope.